cypress/e2e/laboratory-environment-variables.cy.ts

```
@@ -0,0 +1,84 @@
+    expect(window.localStorage.getItem(environmentVariablesStorageKey.global)).equals(null);
+    // cy.dataCy('preflight-editor-mini').should('have.text', '');
+  });
+
+  it('if state just global value, shows that', () => {
+    window.localStorage.setItem(environmentVariablesStorageKey.global, data.envarsJson);
+    visitTargetDevelopment();
+    openPreflightTab();
+    cy.contains(data.envarsJson);
+  });
+
+  it.only('if state just scoped value, shows that', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)